sdk/Lusid.Sdk/Utilities/ApiExceptionExtensions.cs

```diff
@@ -61,11 +64,26 @@ public static LusidProblemDetails ProblemDetails(this ApiException ex)
                 return null;
             }
 
-            //    ApiException.ErrorContent contains a JSON serialized ProblemDetails
-            return JsonConvert.DeserializeObject<LusidProblemDetails>(ex.ErrorContent.ToString(), new JsonConverter[]
+            try
             {
-                new PropertyBasedConverter(),
-            });
+                // ApiException.ErrorContent contains a JSON serialized ProblemDetails
+                return JsonConvert.DeserializeObject<LusidProblemDetails>(
+                    ex.ErrorContent.ToString(), new PropertyBasedConverter()
+                );
+            }
+            catch (JsonException)
+            {
+                return new LusidProblemDetails(
+                    name: "Invalid JSON error content response",
+                    errorDetails: new List<Dictionary<string, string>>
+                    {
+                        new Dictionary<string, string>
+                        {
+                            {"errorContent", ex.ErrorContent.ToString()}
+                        }
+                    },
+                    detail: $"The error content response is not deserializable: {ex.ErrorContent}");
```

Comment:
I think in this case the return should be `null` the same as when there is no error content.